canonical-web-and-design/ubuntu.com
# Dynamic product instructions

## Done
Make the services enabling guidelines dynamic based on the personal accounts affordences.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Login
- See your token 
- See that it lists the services you have available
- Click on them to see the commands to enable and disable them

## Issue / Card
Fixes https://github.com/canonical-web-and-design/base-squad/issues/708
